src/common/TALENTS/priest.ts

```diff
@@ -448,13 +453,13 @@ const talents = createTalentList({
   VOID_ERUPTION_TALENT: {
     id: 228260,
     name: 'Void Eruption',
-    icon: 'spell_priest_void_blast',
+    icon: 'spell_priest_void-blast',
     maxRanks: 1,
   },
   UNFURLING_DARKNESS_TALENT: {
     id: 341273,
     name: 'Unfurling Darkness',
-    icon: 'spell_priest_shadow_mend',
+    icon: 'spell_priest_shadow-mend',
```

Comment:
This should still be `spell_priest_shadow_mend`.
```suggestion
    icon: 'spell_priest_shadow_mend',
```

Reply:
I've undone all the changes to TALENTS